src/main/scala/org/broadinstitute/dsde/firecloud/service/DataUseRestrictionSupport.scala

```diff
@@ -209,7 +208,7 @@ trait DataUseRestrictionSupport extends LazyLogging {
     }.toSeq
 
 
-    val dsLabels:Seq[String] = (attributes.get(AttributeName.withDefaultNS(DS)) collect {
+    val dsLabels:Seq[String] = (attributes.get(AttributeName.withDefaultNS(ConsentCodes.DSURL)) collect {
```

Comment:
This is probably the right thing to do, but I think it needs a corresponding change to `getDiseaseNames` below to not prepend `DataUse.doid_prefix` when calling `OntologyDao.search`.